src/tool/Model/Configuration.cs

```diff
@@ -254,7 +252,6 @@ public bool ParseFromBufferV1(byte[] buffer)
 				MaxSpeedKph = br.ReadByte();
 
 				UseSpeedSensor = br.ReadBoolean();
-				UseDisplay = br.ReadBoolean();
```

Comment:
You will break backwards compatibility here by removing the ReadBoolean() part since it consumes one byte from the stream and repositions. Same for V2. V3 is not released yet so that is fine.

I would probably have left it as:
/* UseDisplay = */ br.ReadBoolean();

Reply:
Ahh shoot good catch, will fix.